addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16412,7 +16412,7 @@ msgstr ""
 #. Description of setting #14102 Settings -> Video -> Discs -> Blu-ray playback mode
 #: system/settings/settings.xml
 msgctxt "#37031"
-msgid "Specifies how Blu-rays should be opened/played back. Disc menus are not fully supported yet and may cause problems."
+msgid "Specifies how Blu-ray's should be opened/played back. Disc menus are not fully supported yet and may cause problems."
```

Comment:
I changed CDs DVDs Blu-rays vs CD's DVD's and Blu-ray's . use of the possessive as determiners is not wrong.